wallet/wallet.go

```diff
@@ -493,6 +506,85 @@ func (w *Wallet) waitUntilBackendSynced(chainClient chain.Interface) error {
 	}
 }
 
+// locateBirthdayBlock returns a block that meets the given birthday timestamp
+// by a margin of +/-2 hours. This is safe to do as the timestamp is already 2
+// days in the past of the actual timestamp.
+func locateBirthdayBlock(chainClient chain.Interface,
```

Comment:
method should have some tests

Reply:
There are some tests within `wallet/chainntnfs_test.go` that already exercise this behavior: `TestBirthdaySanityCheckHigherEstimate` and `TestBirthdaySanityCheckLowerEstimate`. Do you think additional tests would still be nice to have?

Reply:
Looks like they call into `birthdaySanityCheck`. Since they probably cover most of the logic of this method anyway, I agree that we don't have to add extensive tests for this method. But would be nice to have a simple test targeting this method directly, just as a starting point for adding more tests down the line.

Reply:
reminder to add a simple unit test for this method.

Reply:
Done.